File: FR_14_04_Karty/KARTY2.rb
def wynik(x)
    k="23456789TJQKA"
    return k.index(x)+2
end
j=gets()
s=gets()
wj = 0
ws = 0
j.chomp.split('').each do |x|
    wj=wj+wynik(x)
end
s.chomp.split('').each do |x|
    ws=ws+wynik(x)
end

if wj>ws 
    print "JASIO"
elsif wj<ws
    print "STASIO"
else
    print "REMIS"
end

File: FR_14_04_Karty/KARTY2.py
def punkty(x):
    karty="23456789TJQKA"
    return karty.index(x)+2

j=input()
wj=0
s=input()
ws=0

for x in j:
    wj+=punkty(x)

for x in s:
    ws+=punkty(x)

if wj>ws:
    print("JASIO")
elif wj<ws:
    print("STASIO")
else:
    print("REMIS")